libs/metrics/test_positivity.py

```diff
@@ -56,6 +56,46 @@ def _append_variable_index_level(df: pd.DataFrame, field_name: FieldName) -> Non
     )
 
 
+def _make_result_dataset(
```

Comment:
fwiw I think `output` might be more clear here.  I'm finding this function a bit difficult to understand what's coming out - taking me a while to translate pandas operations into the resulting dataframe.